docs/benchmarking-workflow-corefx.md

```diff
@@ -262,4 +262,11 @@ Sample project file change:
 
 Because the benchmarks are not in the CoreFX repository you must do two PR's.
 
-The first thing you need to do is send a PR with the new API to the CoreFX repository. Once your PR gets merged and a new NuGet package is published to the CoreFX NuGet feed, you should remove the Reference to a `.dll` and install/update the package consumed by [MicroBenchmarks](../src/benchmarks/micro/MicroBenchmarks.csproj). After that, you should send a PR to the performance repo.
+The first thing you need to do is send a PR with the new API to the CoreFX repository. Once your PR gets merged and a new NuGet package is published to the CoreFX NuGet feed, you should remove the Reference to a `.dll` and install/update the package consumed by [MicroBenchmarks](../src/benchmarks/micro/MicroBenchmarks.csproj). You can do this by running the following script locally:
+
+```cmd
+py.exe scripts\benchmarks_ci.py --filter $YourFilter -f netcoreapp3.0 --incremental no --architecture x64
```

Comment:
nit: I would remove the `py.exe` (Windows only) part from this command example, and leave it as:
```cmd
$Python3 ./scripts/benchmarks_ci.py --filter $YourFilter -f netcoreapp3.0 --incremental no --architecture x64
```